--- fftw.h	Mon Nov 16 17:39:56 1998
+++ /usr/local/include/fftw.h	Sun Nov 15 10:11:32 1998
@@ -293,13 +293,11 @@
 					   fftw_complex *in, int istride,
 					 fftw_complex *out, int ostride);
 #define FFTW_HAS_PLAN_SPECIFIC
-#ifndef FFTW_NOPROTO
 extern fftw_plan fftw_create_plan(int n, fftw_direction dir, int flags);
-extern void fftw(fftw_plan plan, int howmany, fftw_complex *in, int istride,
-		 int idist, fftw_complex *out, int ostride, int odist);
-#endif
 extern void fftw_print_plan(fftw_plan plan);
 extern void fftw_destroy_plan(fftw_plan plan);
+extern void fftw(fftw_plan plan, int howmany, fftw_complex *in, int istride,
+		 int idist, fftw_complex *out, int ostride, int odist);
 extern void fftw_one(fftw_plan plan, fftw_complex *in, fftw_complex *out);
 extern void fftw_die(const char *s);
 extern void *fftw_malloc(size_t n);
@@ -321,13 +319,10 @@
  * define this symbol so that users know we are using a version of FFTW
  * with wisdom
  */
-#ifndef FFTW_NOPROTO
-extern void fftw_export_wisdom(void (*emitter) (char c, void *), void *data);
-extern fftw_status fftw_import_wisdom(int (*g) (void *), void *data);
-#endif	
-	
 #define FFTW_HAS_WISDOM
 extern void fftw_forget_wisdom(void);
+extern void fftw_export_wisdom(void (*emitter) (char c, void *), void *data);
+extern fftw_status fftw_import_wisdom(int (*g) (void *), void *data);
 extern void fftw_export_wisdom_to_file(FILE *output_file);
 extern fftw_status fftw_import_wisdom_from_file(FILE *input_file);
 extern char *fftw_export_wisdom_to_string(void);
